qark/plugins/webview/helpers.py

```diff
@@ -22,6 +22,10 @@ def valid_set_method_bool(method_invocation, str_bool, method_name="setAllowFile
     :return: Whether the MethodInvocation matches the method name and has the `str_bool` value as its first argument.
     :rtype: bool
     """
+    if len(method_invocation.arguments) == 0:
```

Comment:
Instead of this check we can do the following which should be nicer:
```python
arg = method_invocation.arguments.get(0)
if not isinstance(method_invocation.arguments[0], Literal):
    return False
...
```
And then later in the check make sure we just that variable to refer to the argument. Thanks for the fix.

Reply:
Thx for replying! The list objects in Python do not have a `get` method. I used another way to check this.